### ISSUE TITLE
imread specify channels

### ISSUE THREAD

**Issue by maiskovich** [cgohlke/tifffile#244]
Hello,

Thank you for the great package.

I am using tifffile to load images as part of a PyTorch DataLoader. I have tiff images with more than 30 channels, but I only load 3 of those at the reading time. Currently, in my data loader I load the whole image with tifffile and then use numpy to select the desired channels. I noticed that this operation is taking a lot of time, and I wondered if there is a way to load the desired channels directly with the `imread` method. I checked the code and documentation, and I see ways to specify the page in the imread method, but I don't see a mention of channels.

Is there a way to load specific channels using the `imread` method?

This is how I am currently doing it with numpy:
```  
def __getitem__(self, idx):
        if torch.is_tensor(idx):
            idx = idx.tolist()
        im_tiff = tifffile.imread(self.tiff_files[idx],
                                  maxworkers=32)
        sample = torch.from_numpy(im_tiff[self.channels,:,:]/info.max).float()
```

Any advice to optimise this operation is highly appreciated.

Thank you!

**Comment by cgohlke** [comment]
Did you try the [selection parameter](https://github.com/cgohlke/tifffile/blob/2b5a5208008594976d4627bcf01355fc08837592/tifffile/tifffile.py#L1166-L1174)? It is an index into a zarr array mapped to the image data in the TIFF file. This assumes the image shape and channel axis/dimension/index are known. See the [tests](https://github.com/cgohlke/tifffile/blob/2b5a5208008594976d4627bcf01355fc08837592/tests/test_tifffile.py#L13238-L13312) for examples.

The optimal way depends on how the channels are stored in the TIFF file. They could be found contiguous samples, separate samples, or separate pages, or part of a hyperstack. There are different methods to access these directly with tifffile. 

See also comments at https://github.com/cgohlke/tifffile/issues/200#issuecomment-1613348047 and https://github.com/cgohlke/tifffile/issues/224#issuecomment-1701784550.

**Comment by maiskovich** [comment]
Thank you, @cgohlke; I ended up using the key parameter and managed to filter channels from there. I was confused as the parameter mentioned the concept of `page`, and I thought this referred to a different type of file, but it worked perfectly for my use case. Thank you
